inc/log.class.php

```diff
@@ -279,6 +282,13 @@ static function showForItem(CommonDBTM $item, $withtemplate = 0) {
          return;
       }
 
+      // CSV export link
+      $href = "../front/log/export.php?" . http_build_query([
```

Comment:
```suggestion
      $href = $CFG_GLPI['root_doc'] . "/front/log/export.php?" . http_build_query([
```

will require to declare `global $CFG_GLPI;`